cmd/oras/root/push.go

```diff
@@ -69,7 +69,7 @@ Example - Push multiple files with different media types:
   oras push localhost:5000/hello:v1 hi.txt:application/vnd.me.hi bye.txt:application/vnd.me.bye
 
 Example - Push file "hi.txt" with config type "application/vnd.me.config":
-  oras push --artifact-type application/vnd.me.config localhost:5000/hello:v1 hi.txt
+  oras push --image-spec v1.0 --artifact-type application/vnd.me.config localhost:5000/hello:v1 hi.txt
```

Comment:
This seems to be the only example using `--artifact-type`. Do we need another example without `--image-spec v1.0`? Something like

```
Example - Push file "hi.txt" with artifact type "application/vnd.me.artifact":
  oras push --artifact-type application/vnd.me.artifact localhost:5000/hello:v1 hi.txt
```

Reply:
added